typify-impl/src/value.rs

```diff
@@ -25,6 +26,7 @@ impl TypeEntry {
         &self,
         type_space: &TypeSpace,
         value: &serde_json::Value,
+        super_scoped: bool,
```

Comment:
What would you think about making this into a `scope: TokenStream`?

Reply:
Reduces code duplication, it's a fair point, I'll get that done.